Libraries/Components/WebView/WebView.android.js

```diff
@@ -137,6 +137,16 @@ class WebView extends React.Component {
     scalesPageToFit: PropTypes.bool,
 
     /**
+     * Function that is invoked when a defined URL-scheme has been blocked.
+     */
+    onUrlSchemeBlocked: PropTypes.func,
+
+    /**
+     * An array defining blacklisted URL-schemes.
+     */
+    urlSchemeBlacklist: PropTypes.array,
```

Comment:
As I'm not a native English speaker I was quite unsure about naming here. If "rejected" sounds better I will change it! 👍